client/src/app/contratos/info-contrato/info-contrato.component.ts

```diff
@@ -136,6 +136,18 @@ export class InfoContratoComponent implements OnInit {
     return cor(valor);
   }
 
+  isItemAtipico(item, alertas) {
+    let isAtipico = false;
+    alertas.forEach( alerta => {
```

Comment:
Acho que pode ter uma checagem aqui pra caso o alertas for diferente de null.
Tá dando esse erro em contratos que não tem itens atípicos:
`ERROR TypeError: alertas is null`